.github/CODEOWNERS

```diff
@@ -1,2 +1 @@
-# This should match the owning team set up in https://github.com/orgs/opensearch-project/teams
-*   @opensearch-project/anomaly-detection
```

Comment:
what is the reason for removing this team alias?

Reply:
Based on the proposal for how we will now maintain permissions across the org here: https://github.com/opensearch-project/.github/issues/125 (step 5 of `What will it take to execute?` specifically mentions this). It looks like we are moving away from team based permissions and repo permissions will now only contain individual aliases as collaborators with maintain rights, admin, and triage teams.

Reply:
Got it, thanks for the clarification!